lib/src/main/kotlin/io/bloco/faker/helpers/StringHelper.kt

```diff
@@ -0,0 +1,58 @@
+package io.bloco.faker.helpers
+
+import java.text.Normalizer
+import java.util.*
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+class StringHelper {
+    interface StringReplacer {
+        fun replaceWith(matcher: Matcher): String
+    }
+
+    fun replaceMethod(input: String, regex: String, replacer: StringReplacer): String {
```

Comment:
maybe change this to make use of invokes and extensions... better code. 
avoids the excess verbose on the tests